cognite/client/data_classes/simulators/runs.py

```diff
@@ -232,6 +233,145 @@ def _load(cls, resource: dict[str, Any], cognite_client: CogniteClient | None =
         )
 
 
+class SimulationValueBase(CogniteObject):
+    """
+    Base class for simulation values. This class is used to define the value and its type.
+    The value can be a string, double, array of strings or array of doubles.
+    The maximum length is 1024 for strings and 200 for arrays.
+    """
+
+    def __init__(
+        self,
+        reference_id: str,
+        value: str | int | float | list[str] | list[int] | list[float],
+        value_type: Literal["STRING", "DOUBLE", "STRING_ARRAY", "DOUBLE_ARRAY"],
+        unit: SimulationValueUnitName | None = None,
+        simulator_object_reference: Any | None = None,
```

Comment:
Can we use something more specific than `Any`?
```suggestion
        simulator_object_reference: dict[str, object] | None = None,
```